pkg/envoy/envoy.go

```diff
@@ -230,6 +235,7 @@ func StartEnvoy(adminPort uint32, stateDir, logPath string, baseID uint64) *Envo
 					}
 				}
 				close(e.errCh)
+				logWriter.Close()
```

Comment:
Is it safe to close the logWriter twice? (Check line 168)

Reply:
Good point, I'll remove the new explicit `Close()` calls, and I need to move the `defer logWriter.Close()` after creating the piper.